pure.zsh

```diff
@@ -153,8 +153,8 @@ prompt_pure_preprompt_render() {
 	if [[ -n $prompt_pure_git_arrows ]]; then
 		preprompt_parts+=('%F{$prompt_pure_colors[git:arrow]}${prompt_pure_git_arrows}%f')
 	fi
-	# Git stash symbol (if opted in).
-	if [[ -n $prompt_pure_vcs_info[stash] ]]; then
+	# Git stash symbol (if opted in and not during action).
+	if [[ -n $prompt_pure_vcs_info[stash] && -z $prompt_pure_vcs_info[action] ]]; then
```

Comment:
The reason the previous implementation didn't work is that when we're rebasing with stash disabled, the contents of `$prompt_pure_vcs_info[stash]` is `no patch applied`.

I haven't delved deeper into the meaning of that but with `%m` (misc) present, some other hooks seem to be running as well.